protocol/avdl/chat1/common.avdl

```diff
@@ -187,6 +187,12 @@ protocol common {
     bytes k; // Verifying key
   }
 
+  record SignEncryptedData {
+    int   v; // version = 1
+    bytes b; // signEncryptedData (output of signencrypt.SealWhole)
+    bytes n; // nonce
```

Comment:
Yeah I have a comment below about that validation bit, so I agree with you here. Not sure whether exclusive ownership comes into it though. (I don't think we ever take a chat message in the abstract and ask "who wrote this?". Instead the server tells us "here's a message from foo" and we confirm that that's true. If that's because foo deliberately picked weak keys so that you could ascribe any message to them...well, that's kinda the same as foo just publishing their private keys anyway.)